apis/i18n.d.ts

```diff
@@ -0,0 +1,46 @@
+import { csn, entity } from './cds'
+
+
+declare class I18nBundle {
+  constructor(options: I18nFilesOptions)
+  for(key: number | string | object, locale?: string | object, args?: object): string | undefined
+  at(key: number | string | object, locale?: string | object, args?: object): string | undefined
+  file: string
```

Comment:
if I am reading [this](https://cap.cloud.sap/docs/node.js/cds-i18n#file) correctly, `.file` belongs to `cds.i18n`, rather than `I18nBundle`. Are these two superimposed on purpose?

Reply:
Removed the property.